### ISSUE TITLE
Launcher Not Setting Class in Co-op

### ISSUE THREAD

**Issue by DistinctSoul** [Forevener/DRPGSEL#4]
Hello Forevener. I have recently encountered a problem with the launcher. When running a multiplayer session using RPG+RL, setting my class in the dropdown list of the launcher will not actually take affect when spawning in the game. This also goes for anyone joining. Instead, everyone will spawn in as the marine class, which I take is the default class. While the "Custom Commands" box does allow for a workaround, it would be nice if the dropdown worked, as it would make running a multiplayer session much more intuitive, due to not having to search up how to set your class in a command line instead.

**Comment by Forevener** [comment]
Hello DistinctSoul. I can't replicate this behaviour at the moment. Could you send me a screenshot of the launcher after you made all desired changes and clicked 'Show entire command-line', please?

**Comment by DistinctSoul** [comment]
My sincere apologies. It appears that this problem was due to a wad I had added, which I wouldn't think have any effect on setting the player's class in the launcher, as all it did was add a music box, as well as a bunch of new music. Removing it resolved the issue. However, also re-adding it, and moving it to the top of the load order, resolved the issue, while allowing me to play with the aforementioned wad as well.
